### PR TITLE
Added create and destroy alias how-tos

### LINKED ISSUE #413: Create Alias Outputs with related docs
## Description

Create examples for the `Alias Output` section. It should contain:
- [ ] Create 
- [ ] Governance Transitions
- [ ] State Transitions
- [ ] Unlock Alias Funds
- [ ] Destroy

## Are you planning to do it yourself in a pull request?

Yes/No.

### PR DESCRIPTION
**Pull request by anistark**
# Description of change

Added create and destroy alias how-tos

## Links to any relevant issues

Issue #413 

## How the change has been tested

Run the code example in how_tos

## Change checklist

Tick the boxes that are relevant to your changes, and delete any items that are not.

- [x] I have followed the contribution guidelines for this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: bindings/nodejs/examples/how_tos/alias/create.ts [hunk 1]
```diff
@@ -0,0 +1,61 @@
+// Copyright 2023 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+import { Utils, Wallet, initLogger } from '@iota/sdk';
```
Comment (Thoralf-M) [inline bindings/nodejs/examples/how_tos/alias/create.ts]: 'Utils' is defined but never used

**Review by Thoralf-M — COMMENTED** [review]
The .mdx docs will now only be added in the wiki repo?

File: bindings/nodejs/examples/how_tos/alias/destroy.ts [hunk 2]
```diff
@@ -1,20 +1,25 @@
...
-
-// In this example we will try to destroy the first alias there is in the account. This is only possible if possible
-// foundry outputs have circulating supply of 0.
-//
-// Make sure that `example.stronghold` and `example.walletdb` already exist by
-// running the `how_tos/accounts-and-addresses/create-wallet` example!
-//
-// Rename `.env.example` to `.env` first, then run
-// yarn run-example ./wallet/16-destroy-alias.ts
+import { Utils, Wallet, initLogger } from '@iota/sdk';
```
Comment (Thoralf-M) [inline bindings/nodejs/examples/how_tos/alias/destroy.ts]: 'Utils' is defined but never used

File: bindings/nodejs/examples/how_tos/alias/destroy.ts [hunk 3]
```diff
@@ -1,20 +1,25 @@
 // Copyright 2023 IOTA Stiftung
 // SPDX-License-Identifier: Apache-2.0
 
-import { getUnlockedWallet } from './common';
-
-// In this example we will try to destroy the first alias there is in the account. This is only possible if possible
```
Comment (Thoralf-M) [inline bindings/nodejs/examples/how_tos/alias/destroy.ts:6]: Why don't keep the description?

File: bindings/python/examples/how_tos/alias/destroy.py [hunk 4]
```diff
@@ -0,0 +1,26 @@
+from iota_sdk import Wallet
+from dotenv import load_dotenv
+import os
+
+load_dotenv()
+
+# In this example we will destroy an alias output
+
+# TODO: replace with your own values.
+ALIAS_ID = "0x982667c59ade8ab8a99188f4de38c68b97fc2ca7ba28a1e9d8d683996247e152"
```
Comment (Thoralf-M) [inline bindings/python/examples/how_tos/alias/destroy.py]: In the TS and Rust example we just get the first alias id from the balance

File: bindings/python/examples/how_tos/alias/destroy.py [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+
+wallet = Wallet('./alice-database')
+
+account = wallet.get_account('Alice')
+
+# Sync account with the node
+response = account.sync()
```
Comment (Thoralf-M) [inline bindings/python/examples/how_tos/alias/destroy.py]: So something like this
```suggestion
balance = account.sync()

# We try to destroy the first alias in the account
alias_id = balance['foundries'][0]
```

File: sdk/Cargo.toml [hunk 6]
```diff
@@ -618,12 +618,12 @@ required-features = [ "wallet", "stronghold" ]
...
 name = "create_alias"
-path = "examples/wallet/create_alias.rs"
+path = "examples/how_tos/alias/create.rs"
 required-features = [ "stronghold" ]
 
 [[example]]
 name = "destroy_alias"
-path = "examples/wallet/16_destroy_alias.rs"
+path = "examples/how_tos/alias/destroy.rs"
 required-features = [ "wallet", "stronghold" ]
```
Comment (Thoralf-M) [inline sdk/Cargo.toml]: Please move it to the other how_tos/alias examples at L220

**Review by thibault-martinez — CHANGES_REQUESTED** [review]
Conflicts

File: bindings/python/examples/how_tos/alias/destroy.py [hunk 7]
```diff
@@ -6,21 +6,21 @@
...
 wallet = Wallet('./alice-database')
 
 account = wallet.get_account('Alice')
 
 # Sync account with the node
-response = account.sync()
+balance = account.sync()
+
+# We try to destroy the first alias in the account
+alias_id = balance['foundries'][0]
```
Comment (Thoralf-M) [inline bindings/python/examples/how_tos/alias/destroy.py]:
```suggestion
alias_id = balance['aliases'][0]
```

**Review by Thoralf-M — CHANGES_REQUESTED** [review]
Remove the documentation folder again

File: bindings/python/examples/how_tos/alias/create.py [hunk 8]
```diff
@@ -0,0 +1,23 @@
...
+account.sync()
+
+if 'STRONGHOLD_PASSWORD' not in os.environ:
+    raise Exception(".env STRONGHOLD_PASSWORD is undefined, see .env.example")    
+
+wallet.set_stronghold_password(os.environ["STRONGHOLD_PASSWORD"])
+
+# Send transaction.
+transaction = account.prepare_create_alias_output(None, None).send()
+print(f'Block sent: {os.environ["EXPLORER_URL"]}/block/{transaction["blockId"]}')
```
Comment (Thoralf-M) [inline bindings/python/examples/how_tos/alias/create.py]: Should now work like this
```suggestion
print(f'Block sent: {os.environ["EXPLORER_URL"]}/block/{transaction.blockId}')
```

File: bindings/python/examples/how_tos/alias/destroy.py [hunk 9]
```diff
@@ -0,0 +1,26 @@
...
+alias_id = balance['aliases'][0]
+
+if 'STRONGHOLD_PASSWORD' not in os.environ:
+    raise Exception(".env STRONGHOLD_PASSWORD is undefined, see .env.example")
+
+wallet.set_stronghold_password(os.environ["STRONGHOLD_PASSWORD"])
+
+# Send transaction.
+transaction = account.prepare_destroy_alias(alias_id).send()
+print(f'Block sent: {os.environ["EXPLORER_URL"]}/block/{transaction["blockId"]}')
```
Comment (Thoralf-M) [inline bindings/python/examples/how_tos/alias/destroy.py]:
```suggestion
print(f'Block sent: {os.environ["EXPLORER_URL"]}/block/{transaction.blockId}')
```

File: sdk/Cargo.toml [hunk 10]
```diff
@@ -229,6 +229,16 @@ name = "alias_wallet_transaction"
 path = "examples/how_tos/alias_wallet/transaction.rs"
 required-features = [ "rocksdb", "stronghold" ]
 
+[[example]]
+name = "create_alias"
+path = "examples/how_tos/alias/create.rs"
+required-features = [ "stronghold" ]
```
Comment (Thoralf-M) [inline sdk/Cargo.toml]:
```suggestion
required-features = [ "wallet", "stronghold" ]
```

**Review by Thoralf-M — APPROVED** [review]

**Review by thibault-martinez — APPROVED** [review]

**Pull request merged**